packages/edit-site/src/components/add-new-template/add-custom-template-modal-content.js

```diff
@@ -36,8 +36,7 @@ function SuggestionListItem( {
 		<Composite.Item
 			render={
 				<Button
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					__next40pxDefaultSize
```

Comment:
This can be seen when you go to `site editor` and go to template, add new and select any template.

**Note:** This would have no impact as it has some styles which are overriden.

| Before | After     |
| :-------- | :------- |
| <img width="1369" alt="custom-template-before" src="https://github.com/user-attachments/assets/425158b1-c3fb-4e87-bb06-d8e020cea77b"> | <img width="1369" alt="custom-template-after" src="https://github.com/user-attachments/assets/78d65d8d-a3a3-4af9-9b1e-8154bc9195fe"> |